http/jsonapi/generator/internal/pay/open-api.json

```diff
@@ -452,14 +452,36 @@
         }
       }
     },
-    "/beta/transaction": {
+    "/beta/transaction/{pathDecimal}": {
```

Comment:
Why would we have a decimal in the transaction path?

Reply:
That's just a test case for the generator and does not reflect a real api.